src/shared/components/organisms/Sidebar/atoms/SidebarTreeForm.tsx

```diff
@@ -19,12 +19,13 @@ const SidebarTreeForm = ({
   createCallback,
 }: SideNavigatorFolderFormProps) => {
   const [sending, setSending] = useState<boolean>(false)
-  const [name, setName] = useState<string>('')
+  const [name, setName] = useState<string>('Untitled')
```

Comment:
We might want to consider having this sent by the mappers? So they can choose their own initial name/value, or does it make sense to leave it like this inside shared component?